backend/fms_core/serializers.py

```diff
@@ -209,7 +211,7 @@ class Meta:
                   'individual_id', 'sex', 'pedigree', 'mother_name', 'father_name',
                   'current_volume', 'concentration', 'collection_site', 'tissue_source', 'creation_date', 'phenotype',
                   'depleted', 'coordinates',
-                  'comment')
+                  'comment', 'projects')
```

Comment:
i would insert projects before the comment for readability (comments tend to be long).